app/src/renderer/vuex/modules/wallet.js

```diff
@@ -100,7 +100,7 @@ export default ({ node }) => {
         transactions.map(async t => {
           let blockMetaInfo = await dispatch("queryBlockInfo", t.height)
           t.time = blockMetaInfo && blockMetaInfo.header.time
-          return transactions
+          return t
```

Comment:
i would vote for `transactions` over `t`

Reply:
Adding: I would try to avoid refactoring existing untouched code in PRs so we do not bloat the PRs.

Reply:
ah i see - ok makes sense. thank you for clarifying.